torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -96,6 +96,36 @@ XlaOpVector SizeAdd::Lower(LoweringContext* loctx) const {
   return ReturnOp((input1 + input2), loctx);
 }
 
+SizeSub::SizeSub(torch::lazy::Value a, torch::lazy::Value b)
+    : XlaNode(
+          torch::lazy::OpKind{c10::Symbol::fromQualString("aten::size_sub")},
```

Comment:
Well, I am ok with either way as long as it runs, but if you want to update this for SizeSub, you need to update this for all nodes including `SizeAdd` above.

Reply:
From the code, it looks like c10::Symbol would just assign any unseen string an incremental ID. I guess the problem I'm having is that there is no straightforward way you can do a comparison later on. For any aten ops, you can simply do:
```
op == at::aten::sub
```

For any custom op, I guess you have to do:
```
op == c10::Symbol::fromQualString("aten::size_sub")
```

Unless you could define an enum like structure in our end. So what benefits do you think we get from aten::size_sub?

Reply:
It seems semantically cleaner to me: `at::aten::sub` feels like we are doing subtraction between 2 tensors, but here it's between 2 shape dimension (eg tensor.shape[0]). wdyt?

Reply:
From the code perspective, yes. But I wonder if this helps anything practical for instance, LazyIR dump or HLO dump.